index.js

```diff
@@ -10,13 +10,18 @@ function over(e) {
 }
 
 
+/**
+ * @param {DragEvent} e
+ */
```

Comment:
please drop these comments (and also below), they are not useful here

Reply:
The JSDoc allows for typescript language server to infer the type when being ran on JavaScript

Reply:
yes, but the rest of this code doesn't follow that pattern, and the code doesn't interact with any other TS code, so IMO it's better to keep it clean.